internal/sdkprovider/service/pg/pg_user.go

```diff
@@ -128,6 +124,23 @@ func resourcePGUserUpdate(ctx context.Context, d *schema.ResourceData, m interfa
 		return diag.FromErr(err)
 	}
 
+	if d.HasChange("pg_allow_replication") {
```

Comment:
Can we somehow combine previous `client.ServiceUsers.Update(projectName, serviceName, username,
		aiven.ModifyServiceUserRequest{...` with this logic? It is strange to see two update calls to the same endpoint one after another. 

So the question is - can `Operation: set-access-control` also update passwords

Reply:
no, it's not possible